src/main/java/sorald/sonar/Checks.java

```diff
@@ -111,7 +111,7 @@ public static String getRuleKey(Class<? extends JavaCheck> checkClass) {
                                         checkClass.getName() + " does not have a key"));
     }
 
-    private static String stripDigits(String s) {
+    public static String stripDigits(String s) {
```

Comment:
Given the suggestion above, this change can be reverted.

On a totally unrelated note: this method does the exact opposite of stripping digits (it strips everything BUT digits). Should be renamed. But that's for another PR :)